Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/AssetCreator.h

```diff
@@ -138,6 +138,7 @@ namespace AZ
             }
             else
             {
+                Data::AssetManager::Instance().AssignAssetData(m_asset);
```

Comment:
Did you confirm the appropriateness of this change with the Helios team?

Reply:
I haven't seen this pattern before but I can't think of a reason it would be an issue.  Seems like it's a decent way to update an asset if some other system already loaded it.